climada/hazard/base.py

```diff
@@ -1887,21 +1871,11 @@ def from_hdf5(cls, file_name):
         hazard_kwargs = dict()
         with h5py.File(file_name, 'r') as hf_data:
             for (var_name, var_val) in haz.__dict__.items():
-                if var_name != 'tag' and var_name not in hf_data.keys():
+                if var_name not in hf_data.keys():
                     continue
                 if var_name == 'centroids':
                     hazard_kwargs["centroids"] = Centroids.from_hdf5(
                         hf_data.get(var_name))
-                elif var_name == 'tag':
```

Comment:
Will this not make it impossible to read files created with previous versions of CLIMADA as the `haz_type` cannot be loaded?